core/arch/arm/plat-ls/conf.mk

```diff
@@ -125,15 +125,8 @@ CFG_CRYPTO_SIZE_OPTIMIZATION ?= n
 # on the command line
 CFG_NXP_CAAM ?= n
 
-ifeq ($(CFG_NXP_CAAM),y)
-# If NXP CAAM Driver is supported, the Crypto Driver interfacing
-# it with generic crypto API can be enabled.
-CFG_CRYPTO_DRIVER ?= y
-CFG_CRYPTO_DRIVER_DEBUG ?= 0
-else
-$(call force,CFG_CRYPTO_DRIVER,n)
+ifeq ($(CFG_NXP_CAAM),n)
```

Comment:
You should only test for `y` or not `y`, `n` is obviously not `y` but so is an empty string.